addon/model/model-text.ts

```diff
@@ -25,6 +27,7 @@ export default class ModelText extends ModelNode {
 
   set content(value: string) {
     this._content = value;
+    this.addDirty('content');
```

Comment:
shouldn't a node be dirty on create as well?

Reply:
look at the constructor of the superclass :)